test/integration/daemonset/daemonset_test.go

```diff
@@ -312,6 +312,41 @@ func validateDaemonSetPodsAndMarkReady(
 	}
 }
 
+func validateDaemonSetPodsActive(
+	podClient corev1client.PodInterface,
+	podInformer cache.SharedIndexInformer,
+	numberPods int,
+	t *testing.T,
+) {
+	if err := wait.Poll(time.Second, 60*time.Second, func() (bool, error) {
+		objects := podInformer.GetIndexer().List()
+		if len(objects) != numberPods {
```

Comment:
```suggestion
		if len(objects) < numberPods {
```

maybe there coudl be other pods?